router_client.go

```diff
@@ -999,3 +1000,35 @@ func (r *routerClient) ResetMissionControl(ctx context.Context) error {
 	)
 	return err
 }
+
+type ChannelStatusAction int32
+
+const (
+	ChanStatusActionENABLE  ChannelStatusAction = 0
```

Comment:
nit: naming wise I'd go for `ChanStatusActionEnable` (same for the rest of the consts)

Reply:
Alternatively we could just pass `routerrpc.ChanStatusAction` to the function as well as this is just an enum.